README.md

```diff
@@ -3,7 +3,7 @@
 [![serverless](http://public.serverless.com/badges/v3.svg)](http://www.serverless.com)
 [![version](https://badge.fury.io/js/serverless-lumigo.svg)](https://www.npmjs.com/package/serverless-lumigo)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
-[![CircleCI](https://circleci.com/gh/lumigo-io/serverless-lumigo-plugin.svg?style=svg)](https://circleci.com/gh/lumigo-io/serverless-lumigo-plugin)
+[![CircleCI](https://circleci.com/gh/lumigo-io/serverless-lumigo-plugin.svg?style=svg)](https://circleci.com/gh/lumigo-io/serverless-lumigo-plugin) 
```

Comment:
```suggestion
[![CircleCI](https://circleci.com/gh/lumigo-io/serverless-lumigo-plugin/tree/master.svg?style=svg&circle-token=a136efcbcc581c23e081c4fa171a2d05c6fb8ab3)](https://circleci.com/gh/lumigo-io/serverless-lumigo-plugin/tree/master)
```

Reply:
I think it's because the circleci project is private. You need to add a token to actually view the badge

Reply:
wait, why is the circleci project private? can we make it public? it should be public